### ISSUE TITLE
[FEATURE] User support

### ISSUE THREAD

**Issue by jeff-horton-ho-sas** [UKHomeOffice/engineering-guidance-and-standards#363]
Request for discussion on what level of support we can / want to offer for engineers using the site. 

**What is the feature or enhancement you are suggesting?**
Provide a standard way for an engineer to request support using the site, or applying a principle, standard, or pattern.

**What would be the benefit of doing this?**
Makes the site more accessible and helps identify where content is unclear

**Additional information**
This was highlighted when discussing WCAG2.2's new clause 3.2.6 Consistent help in #361. Any implementation will need to display how to get help in a consistent place on each page.

Currently the site has the "New service" banner on each page with a link to [a page on providing feedback](https://engineering.homeoffice.gov.uk/provide-feedback/), which provided the option of raising an issue with the feedback in this repository, or emailing [segas@digital.homeoffice.gov.uk](mailto:segas@digital.homeoffice.gov.uk). There is also a link to the [accessibility statement](https://engineering.homeoffice.gov.uk/accessibility-statement/#reporting-accessibility-problems-with-this-website) in the footer displayed on each page, which has a section on reporting accessibility issues with the same points of contact.

**Please confirm the below**

- [X] I have checked through the issues on the repository and this has not already been suggested

**Questions for discussion**
* Is the current feedback link + accessibility link sufficient?
* If not: do we modify the existing feedback link and page to communicate it can be used for feedback or support, or create a separate link and page for getting support as well as the current feedback link? Something else (suggestions below)?
* Should links to the various Slack channels associated with SEGAS be included as additional contact points on the feeback page or support page if we add one? (i.e. \#segas-support, \#segas-site, and the various \#segas-\<guild name> channels) 
* With the current setup responding to feedback/accessibilityGitHub issues and/or emails falls to the members of engineering community who volunteer to be part of the SEGAS guilds and/or help out with the site. 
	* Are we ok with the current commitment that esp. accessibility issues will be responded to quickly? 
	* Is it still possible if we open this up to more general help applying the principles/standards/patterns? 
	* Do we need to put anything in place to make sure this continues to be the case?
